tests/unit/LocalDB.js

```diff
@@ -253,9 +253,10 @@ define([
 					]);
 				});
 			},
-			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))":
+			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))": function() {
```

Comment:
You are right, that actually revealed a different issue when I fixed it, and that was also fixed by using `Object.create` in `Store`.